### PR TITLE
Create Product “ce/developer-board-4-version-3/README”

### PR DESCRIPTION
**Pull request by linaro-test-account**
Automatically generated by Netlify CMS

### CODE REVIEW

File: _product/ce/developer-board-4-version-3/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+product_short_desc: >-
+  The Developer Board 4 Version3 is a 96Boards compliant community board based
+  on Qualcomm® Snapdragon Quad-core ARM Cortex-A53.
+keywords: >-
+  Developer Board, Wi-Fi, Bluetooth connectivity, development, board, low cost,
+  Product, Development, Platform, Qualcomm, high-speed connector,  functional
+  modules
+dateAdded: '2020-01-23 04:13:57'
+product_images: images/96boards-banner-project-2.jpg
+image: /assets/images/content/96Boards-Logo_v-Consumer.png
```
Comment (ric96) [inline _product/ce/developer-board-4-version-3/README.md]: wrong image path.
You have added the image to  _product/ce/developer-board-4-version-3/images/96Boards-Logo_v-Consumer.png but you mention /assets/images/content/96Boards-Logo_v-Consumer.png

File: _product/ce/developer-board-4-version-3/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    link-price: '129.99'
+    link-price-currency: USD
+    link-title: Sophon Edge (US)
+    link-url: 'https://sophon.ai/product/view/00020181017110652CiJsd2n6aOcHr4a/view.html'
+    type: board
+  - from: Sophon.ai
+    link-price: '899.00'
+    link-price-currency: CNY
+    link-title: Sophon Edge (CN)
+    link-url: 'https://sophon.cn/product/view/00020181017110652CiJsd2n6aOcHr4a/view.html'
```
Comment (ric96) [inline _product/ce/developer-board-4-version-3/README.md]: Please fix this section with DB4 related information.

File: _product/ce/developer-board-4-version-3/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    tab_align_right: false
+    tab_link: /documentation/consumer/sophon-edge/getting-started/
+    tab_title: Getting Started
+  - active: false
+    tab_align_right: false
+    tab_link: /documentation/consumer/sophon-edge/
+    tab_title: Documentation
+  - active: false
+    tab_align_right: true
+    tab_link: 'https://discuss.96boards.org/c/products/sophon-edge/'
```
Comment (ric96) [inline _product/ce/developer-board-4-version-3/README.md]: remove mentions of sophon-edge

File: _product/ce/developer-board-4-version-3/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  - name: CPU
+    value: >-
+      Qualcomm® Snapdragon Quad-core ARM Cortex-A53 64-bit application
+      processors with 512kB L2 cache
+  - name: RAM
+    value: 1GB or 2GB LPDDR3 533MHz
+product_sidebar_sections:
+  - icon: fa-terminal
+    items:
+      - link: /documentation/consumer/sophon-edge/downloads/
```
Comment (ric96) [inline _product/ce/developer-board-4-version-3/README.md]: remove mentions of sophon-edge

**Comment by kylekirkby** [comment]
@ric96 - I created this PR using the NetlifyCMS instance that I had been setting up. I've currently put this on hold since the Connect automation work is a higher priority right now. As soon as I've made some more progress on this, I'll get back to writing a guide on how to use NetlfiyCMS for 96Boards and iron out any bugs. :) 

Best regards,
Kyle

**Pull request closed without merging**